### PR TITLE
Setting correct icons for About Screen.

### LINKED ISSUE #323: [Assigned] set correct icons for About Screen.
## Overview (Required)
- icons below are not proper ones. adapt correct ones. 

## Links
- #316 
- [Design](https://app.zeplin.io/project/5a327a64f0b20176887c556c/screen/5a5b17ff4d545fa82084b089)
- [Asset](https://app.zeplin.io/project/5a327a64f0b20176887c556c/screen/5a53b2cc33773142c504b949)

### PR DESCRIPTION
**Pull request by ycrack**
## Issue
- close #323 

## Overview (Required)
- Add and apply icons and logo for AboutThisApps
- Prepare to add an icon Medium

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
:cool:

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -77,4 +77,10 @@
         <item name="android:layout_height">match_parent</item>
         <item name="android:fitsSystemWindows">true</item>
     </style>
+
+    <style name="SocialMediaButton" parent="Theme.AppCompat.Light">
```
Comment (takahirom) [inline app/src/main/res/values/styles.xml]: Thanks 👍 
I think Theme.AppCompat.Light is a theme. So we should not use it for Button. What do you think?

Comment (ycrack) [reply app/src/main/res/values/styles.xml]: I agree with you. This is a mistake. I'll fix this soon!

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="primary">#FBC812</color>
```
Comment (takahirom) [inline app/src/main/res/values/colors.xml]: Thank you 👍 
But, I think this is not related this PR. And the git log gets dirty. 😭 
Please revert this. and If you can please send other pull requests. 🙇

Comment (ycrack) [reply app/src/main/res/values/colors.xml]: Oh, sorry... I'll be careful from now on. Please just a moment while finish cleaning it.

**Comment by ycrack** [comment]
Sorry I made you wait. Adding and Applying has completed! please review... 👍

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (takahirom) [reply app/src/main/res/values/styles.xml]: Can you fix this?

Comment (ycrack) [reply app/src/main/res/values/styles.xml]: Oops, I forgot fix this... I'm sorry to have a trouble many times... 😢

**Comment by takahirom** [comment]
LGTM Thanks!
